server/models/rider.ts

```diff
@@ -6,13 +6,19 @@ export enum Accessibility {
   WHEELCHAIR = 'Wheelchair',
 }
 
+export enum Organization {
+  RED_RUNNER = 'Red_Runner',
+  CULIFT = 'CULift'
+}
+
 export type RiderType = {
   id: string
   firstName: string
   lastName: string
   phoneNumber: string
   email: string
   accessibility: Accessibility[]
+  organization: Organization []
```

Comment:
RedRunner and CULift riders are mutually exclusive. Could you make it so that the field is not an array, enforcing that you  can only set one or the other?